app/cells/form_cell_base.rb

```diff
@@ -4,6 +4,7 @@ class FormCellBase < Cell::ViewModel
   private
 
   def label
-    options[:label] || controller.t(".#{options[:name]}")
+    return options[:label] if options.key?(:label)
+    controller.t(".#{options[:name].sub(/_id/, '')}")
```

Comment:
Let's make this `/_id\z/` instead, otherwise something like `waffle_identificator_string` gets replaces with `waffleentificator_string` ;)